capture/capture.go

```diff
@@ -422,59 +419,30 @@ func (l *Listener) activatePcapFile() (err error) {
 }
 
 func (l *Listener) setInterfaces() (err error) {
```

Comment:
It has changed a lot comparing to the previous version. Can you explain what changed here?

Reply:
the major functionalities are still the same! I removed the part where we unnecessary store the addresses of very interface